docs/v1/concepts/host-env.mdx

```diff
@@ -33,6 +33,10 @@ Using an environment variable allows you to disable tracking of this data while
 can enable it again when deploying in a production environment.
 
 Opting out is not recommended by AgentOps, as it reduces information used to debug in the agent development process,
-[but AgentOps respects our privacy](mailto:alex@agentops.ai) conscious users and has made opting out available.
+but AgentOps respects our privacy conscious users and has made opting out available.
+
+<Warning>
+[Send us an email](mailto:alex@agentops.ai) with Subject: `Env Data Opt-Out`
```

Comment:
We don't have a way to opt users out right now, except via the SDK setting

Reply:
There was already an implied Call to Action to email Alex before this change lol

Reply:
I'll change it to just focus on the flag

Reply:
Done + broken links fixed